### PR TITLE
#215 support other triple slash comments

### PR DESCRIPTION
**Pull request by ansyral**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    <p>
+      <xsl:apply-templates />
+    </p>
+  </xsl:template>
+
+  <xsl:template match="b">
+    <strong>
+      <xsl:apply-templates />
+    </strong>
+  </xsl:template>
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl]: Why?

Comment (ansyral) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl]: in CAPS, \<b\> would be transformed to \<strong\> eventually. not sure whether there is any css setting in the two tags

**Comment by vwxyzh** [comment]
👍

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj [hunk 2]
```diff
@@ -129,6 +130,9 @@
       <Name>Microsoft.DocAsCode.YamlSerialization</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Include="Transform\TripleSlashCommentTransform.xsl" />
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj]: add resources to project.json, as similar to docfx's project.json

Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Microsoft.DocAsCode.Metadata.ManagedReference.csproj]: You may rename the resource as what docfx.csproj does. The behavior for Resource Name in project.json is different from the one in csproj

---
In reply to: [59156174](https://github.com/dotnet/docfx/pull/249#discussion_r59156174) [](ancestors = 59156174)

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs [hunk 3]
```diff
@@ -39,6 +40,10 @@ private TripleSlashCommentModel(string xml, ITripleSlashCommentParserContext con
         {
             // Normalize xml line ending before load into xml
             XDocument doc = XDocument.Parse(xml, LoadOptions.SetLineInfo | LoadOptions.PreserveWhitespace);
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs]: parse into transformer?

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl (hunk 1 above)
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Transform/TripleSlashCommentTransform.xsl]: differences between <strong> and <b> is that: <strong> tags tell crawlers to treat the inner text as **important**, while <b> is simply a style tag. [http://www.html-5-tutorial.com/strong-and-b-elements.htm](http://www.html-5-tutorial.com/strong-and-b-elements.htm)

---
In reply to: [58978110](https://github.com/dotnet/docfx/pull/249#discussion_r58978110) [](ancestors = 58978110)

**Comment by ansyral** [comment]
updated. @vicancy

File: src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs [hunk 4]
```diff
@@ -36,9 +37,11 @@ public class TripleSlashCommentModel
         public Dictionary<string, string> TypeParameters { get; private set; }
 
         private TripleSlashCommentModel(string xml, ITripleSlashCommentParserContext context)
-        {
-            // Normalize xml line ending before load into xml
-            XDocument doc = XDocument.Parse(xml, LoadOptions.SetLineInfo | LoadOptions.PreserveWhitespace);
+        { 
+            // Transform triple slash comment
+            _transformer = new TripleSlashCommentTransformer(this.GetType().Assembly, "Transform", "TripleSlashCommentTransform.xsl");
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs]: how about passing in Stream to transformer?

Comment (ansyral) [inline src/Microsoft.DocAsCode.Metadata.ManagedReference/Parsers/TripleSlashCommentModel.cs]: why?

**Comment by vicancy** [comment]
:+1 LGTM

**Pull request merged**